www/front_src/src/Resources/columns/Graph/index.tsx

```diff
@@ -25,7 +25,7 @@ import { ColumnProps } from '..';
 const JSXXAxis = (XAxis as unknown) as (props) => JSX.Element;
 const JSXYAxis = (YAxis as unknown) as (props) => JSX.Element;
 
-const graphHeight = 350;
+const graphHeight = 200;
```

Comment:
200 is too small as a height for the graph, the legend can overlap if there are too many elements:

![Screenshot from 2020-03-29 11-27-56](https://user-images.githubusercontent.com/8367233/77845594-6b80b000-71b0-11ea-8ca2-3905ad120d62.png)

Reply:
I think the legend can overlap also with 350 height (eg: monitoring of network interfaces of a switch)
It should be better to make a custom legend to manage lot of metrics

Reply:
Anyway, I've increased the height to 250